### ISSUE TITLE
Vertical Spacing

### ISSUE THREAD

**Issue by ryanmclaughlin** [colbycheeze/bluegrid#10]
Thoughts on vertical spacing? I've come across so many places that would benefit from having something like this. Your demo is a great example.

![image](https://cloud.githubusercontent.com/assets/451112/12869598/30cc06f6-ccd9-11e5-848e-c6ff3d66b79c.png)

**Comment by colbycheeze** [comment]
You are referring to adding bottom margin to columns so that when they wrap, they space themselves out?

If you have a screenshot / example that would help me think through some of the use cases you are talking about and if I might add it to the API

**Comment by ryanmclaughlin** [comment]
I guess it's more about having a class for it than anything. Take a card layout for example: horizontal spacing is easy with gutters, but then you need to add margins for the vertical spacing with rules. Not a big deal, but It'd be nice for prototyping.

**Comment by colbycheeze** [comment]
I think for vertical margins / spacing it would be best to just write that helper class into your own css. I usually have a 'utils/_helpers.scss' for small prototyping things like that. Might confuse / water down the library to have it here.

It's already a bit of a stretch that I decided to put the breakpoints / container stuff within here since I like to keep libs as single responsibility...BUT the goal of the auto-gen classes was to make fast prototyping easier so I suppose I achieved that? :smile:
